site/confirmdelete.html.jet

```diff
@@ -0,0 +1,17 @@
+{{extends "templates/application/application.jet"}}
+
+{{block head()}}
+  {{yield seo(title=i18n("confirm_delete_header"), index=false)}}
+{{end}}
+
+{{block body()}}
+  <main id="main" class="container page form-page">
+    <div class="page-header text-center">
+      <h1>{{ i18n("confirm_delete_header") }}</h1>
+    </div>
+    <s72-confirm-delete-form />
+    <div class="page-content">
```

Comment:
Is this mean to be after the relish component?

Reply:
It was but it isn't showing up that way, I should probably make the template match and forget this ever happened.

Reply:
Looks like my stuff was running off hopes and dreams. I added some css but you probably should double check I haven't done something taboo in there.